BUILD

```diff
@@ -86,7 +86,7 @@ move(
 		"amd64/lib/libthread.a": "lib/libthread.a",
 		"amd64/lib/libusb.a": "lib/libusb.a",
 		"amd64/lib/libventi.a": "lib/libventi.a",
-		"amd64/harvey": "bin/harvey",
+		"amd64/harvey.32bit": "bin/harvey.32bit",
```

Comment:
almost. You might want to just use mine; it puts it in sys/src/9/amd64/harvey.32bit, which is what all the scripts expect and which will work assuming union mounts.